src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -264,12 +271,12 @@ public static IEnumerable<Guid> GetAggregateProjectKinds(IVsHierarchy hierarchy)
 
         public static bool IsKnownTestProject(IVsHierarchy vsProject)
         {
```

Comment:
yep. (in all those cases the exception would be replaced with ObjectNullRefence instead of argument in the worst case, but it should not be different than it was due to the current callers of those APIs).